github-dark.css

```diff
@@ -1529,8 +1529,9 @@
   a.filter-item.selected, .date-selector td.selected,
   .date-selector td.selectable-day:hover, .subnav-item:hover, .subnav-item:focus,
   .owner-select-target:hover, .owner-select-target:focus,
-  .suggester li.navigation-focus, .suggester li.navigation-focus small {
-    color: #efefef !important;
+  .suggester li.navigation-focus, .suggester li.navigation-focus small,
+  .label-select-menu .select-menu-item.selected, .octicon .octicon-x {
```

Comment:
> Also, not sure about this color change below. May I suggest just adding it to #fff instead?

yea definitely.

Im not sure what you mean about fill: currentColor; group, I cant see any anywhere.